src/averageAges.js

```diff
@@ -38,6 +50,19 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const women = (withChildren === undefined)
```

Comment:
withChildren here can be boolean or undefined, so shorthand condition may be used.
```suggestion
  const women = withChildren
```